### ISSUE TITLE
unrecognized class name 'GMenu' when loading menubar from .ui

### ISSUE THREAD

**Issue by KernelDeimos** [gotk3/gotk3#206]
Hello,

I'm trying to add a menubar to an application defined in a file called `app-menu.ui`.

To get the menu object, I'm using the following code snippet:
```go
menuBuilder, err := gtk.BuilderNew()
checkErr(err)
err = menuBuilder.AddFromFile("app-menu.ui")
checkErr(err)
menuO, err := menuBuilder.GetObject("appmenu")
checkErr(err)
```

The `checkErr` function simply prints the error if `err != nil`. Unfortunately, after that last statement I receive the error `unrecognized class name 'GMenu'`, and the menuO object is nil.

Is this an issue, or am I doing something wrong?

**Comment by rtemb** [comment]
Hello!
Any updates with it issue? 
Seems faced with same problem. 
I'm trying to get GtkFixed container and got error: "unrecognized class name 'GtkFixed'"

Part of my code:
```go
gtk.Init(nil)
u.Builder, err = gtk.BuilderNew()
if err := u.Builder.AddFromFile("main.glade"); err != nil {
	log.Fatal(err)
}
mainWindow, err := u.Builder.GetObject("window_main")
if err != nil {
 log.Fatal(err)
}

obj2, err2 := u.Builder.GetObject("submit")
if err2 != nil {
	log.Fatal(err2)
}
u.submitBtn = obj2.(*gtk.Button)

obj3, err3 := u.Builder.GetObject("answer_container")
if err3 != nil {
	log.Fatal(err3)
}
u.answerContainer = obj3.(*gtk.Container)
```
But u.Builder.GetObject("submit") call works fine.
Does anyone know what could be wrong here?

I'm using gtk+3/3.22.30

**Comment by Bios-Marcel** [comment]
@rtemb There are simply no bindings for `GtkFixed`, so I assume that's the problem ;)

You can probably add those quite easily, but if you don't want to, I might do it over the next days.

**Comment by rtemb** [comment]
@Bios-Marcel Yes, you are right. The problem in bindings.
I've checked previously merged MRs with adding bindings... i'm not very experienced in go/c, i think it'll takes a very much time from me to implement it
it would be grate if you can do this.

**Comment by rtemb** [comment]
@Bios-Marcel Thanks a lot. I am very appreciate to you!
Hope next time i'll able to add bindings by my own :)

**Comment by firelizzard18** [comment]
This issue is not actually fixed. @rtemb brought up a separate issue, which was fixed. But `unrecognized class name 'GMenu'` is not fixed as of c6f993830205.

**Comment by firelizzard18** [comment]
For anyone facing this issue (with `GMenu`), until #784 is merged you can fix it by adding the following to `go.mod`:

```
replace github.com/gotk3/gotk3 => github.com/firelizzard18/gotk3 v0.5.3-0.20210605064131-3c8c167048d3
```

This tells Go to use firelizzard18/gotk3@3c8c167048d3 instead of this repo, which fixes this issue.
